sfall_testing/gl_test_metarule_exists.ssl

```diff
@@ -0,0 +1,30 @@
+#include "test_utils.h"
+#include "sfall.h"
+
+procedure start begin
+
+    display_msg("Testing metarule_exist functions...");
+
+    call assertEquals("string_format", metarule_exist("string_format"), 1);
+    call assertEquals("metarule_exist", metarule_exist("metarule_exist"), 1);
+
+    // This will fail on sFall
+    call assertEquals("opcode_exists", metarule_exist("opcode_exists"), 1);
+
+    if metarule_exist("opcode_exists") then begin // And example if how to use it
+        #define opcode_exists(opcode) sfall_func1("opcode_exists", opcode)
+        #define OP_GET_UPTIME 0x81B3
+        #define OP_NOOP 0x8000
+        #define OP_WRITE_INT 0x81D1
```

Comment:
Maybe we could upstream this in sfall so people don't need to add the #define to use it?